src/manifolds/FixedRankMatrices.jl

```diff
@@ -408,3 +407,21 @@ function zero_tangent_vector!(
     X.Vt .= zeros(eltype(X.Vt), k, n)
     return X
 end
+
+@doc raw"""
+    to_ambient(X::SVDMPoint)
+
+Reconstruct a low-rank matrix from the its `SVDMPoint` representation.
+"""
+function to_ambient(X::SVDMPoint)
```

Comment:
This should be called `embed!`, see https://juliamanifolds.github.io/Manifolds.jl/latest/interface.html#ManifoldsBase.embed!-Tuple{Manifold,Any,Any}, where we have two cases:

For a default embedding that one “usually has in mind” one would use as the first parameter just the manifold (`::FixedRankMatrices` obviously has a default embedding in that sense), and for further/other embeddings use an `EmbeddedManifold`, which is “just” a type to bind two manifolds, a Manifold `M` and its embedding (one of the several it might have).

Note that `embed!` works in place of the second parameter, the `embed` then automatically works, too :)

Reply:
I was not aware of this function. I will make the changes tomorrow!

Reply:
That's completely fine that you were not aware (luckily I knew of `to_ambient`), ours is called `embed` since it might be more general then “just” going into ambient space.